frontend/device/remarkable/device.lua

```diff
@@ -118,5 +164,16 @@ function Remarkable:reboot()
     os.execute("systemctl reboot")
 end
 
-return Remarkable
+local f = io.open("/sys/devices/soc0/machine")
+if not f then error("missing sysfs entry for a remarkable") end
 
+local deviceType = f:read("*line") 
```

Comment:
nit: `f:close()`

Reply:
Also since you always have `deviceType` you can log it. We actually do that in other platforms. For instance https://github.com/koreader/koreader/blob/master/frontend/device/android/device.lua#L442-L443 or https://github.com/koreader/koreader/blob/master/frontend/device/sdl/device.lua#L369